app/forms/state_file/az_state_credits_form.rb

```diff
@@ -5,11 +5,26 @@ class AzStateCreditsForm < QuestionsForm
     validates :tribal_wages, presence: true, allow_blank: false, numericality: { only_integer: true, greater_than_or_equal_to: 1 }, if: -> { tribal_member == "yes" }
     validates :armed_forces_wages, presence: true, allow_blank: false, numericality: { only_integer: true, greater_than_or_equal_to: 1 }, if: -> { armed_forces_member == "yes" }
 
+    with_options if: -> { tribal_wages.present? || armed_forces_wages.present? } do
+      validate :exceeds_1040_amount
+    end
+
     def save
       attributes_to_save = attributes_for(:intake)
       attributes_to_save[:tribal_wages] = nil if tribal_member == "no"
       attributes_to_save[:armed_forces_wages] = nil if armed_forces_member == "no"
       @intake.update(attributes_to_save)
     end
+
+    private
+
+    def exceeds_1040_amount
+      amount_limit = @intake.direct_file_data.fed_wages_salaries_tips.to_i
```

Comment:
I think this field might already be an integer, so I'm not sure `.to_i` is necessary, but definitely doesn't break anything.

Reply:
That's a good catch I'll fix this